app/views/onboarding/completion/show.html.erb

```diff
@@ -4,5 +4,46 @@
     <p>
       We'll have a look at school details you've sent us and let you know when your school goes live.
     </p>
+
+    <h3>Sign up to our mailing list to receive Energy Sparks updates</h3>
+    <!-- Begin Mailchimp Signup Form -->
+    <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
+    <style type="text/css">
+      #mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
+      /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
+      We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+    </style>
+    <div id="mc_embed_signup">
+      <form action="https://energysparks.us16.list-manage.com/subscribe/post?u=a132136ec1fca0e5b19fbe63d&amp;id=ed205db324" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```

Comment:
Should we have this as an environment variable rather than being in't code do you think?